astrocut/cube_cut.py

```diff
@@ -31,84 +32,97 @@ class CutoutFactory():
     Class for creating image cutouts.
 
     This class emcompasses all of the cutout functionality.  
-    In the current version this means creating cutout target pixel files from TESS full frame image cubes.
+    In the current version this means creating cutout target pixel files from both 
+    SPOC (Science Processing Operations Center) and TICA (Tess Image CAlibration) 
+    full frame image cubes.
+
     Future versions will include more generalized cutout functionality.
     """
 
     def __init__(self):
         """
-        Initiazation function.
+        Initialization function.
         """
 
-        self.cube_wcs = None  # WCS information from the image cube
-        self.cutout_wcs = None  # WCS information (linear) for the cutout
+        self.cube_wcs = None # WCS information from the image cube
```

Comment:
Having all these whitespace changes are a bit unfortunate. Is there a particular style you are using here? 

`black` style (which will be used in astropy) specifies two spaces between code and comments. If possible, would be nice to revert these changes --- or can just run `black` on it, ideally in a separate PR without any functional changes. 

Same goes for all the changes to the alignment on the `img_kwds`. 

The extra changes make it difficult to review since there's a much larger diff.